core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -264,6 +264,15 @@
    void removeCache(String cacheName);
 
    /**
+    * Removes the cache with the given name from the local node.
+    * <p/>
+    * This operation removes the cache and all its contents from memory and cache stores.
+    *
+    * @param cacheName Name of the cache to remove.
+    */
+   void removeCacheLocally(String cacheName);
```

Comment:
I guess we could expose the method only in `DefaultCacheManager` and cast in `RemoveCacheCommand.perform()`?